src/modules/SourceDialogs.rb

```diff
@@ -2129,17 +2119,17 @@ def SelectWidgetHelp
       # help text
       help_text = _(
         "<p><big><b>Media Type</b></big><br>\n" \
-          "The software repository can be located on CD, on a network server,\n" \
-          "or on the hard disk.</p>"
+        "The software repository can be located on CD, on a network server,\n" \
+        "or on the hard disk.</p>"
       )
 
       # help, continued
       help_text = Ops.add(
```

Comment:
Could we get rid of those annoying `Ops.add`?

Reply:
yes, it is of course possible. But question is where we will stop :)